connectivity/drivers/ble/FEATURE_BLE/TARGET_NORDIC/TARGET_NORDIC_CORDIO/CMakeLists.txt

```diff
@@ -1,8 +1,6 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-if("MCU_NRF52840" IN_LIST MBED_TARGET_LABELS)
-    add_subdirectory(TARGET_MCU_NRF52840)
-elseif("NRF5x" IN_LIST MBED_TARGET_LABELS)
+if("NRF5x" IN_LIST MBED_TARGET_LABELS)
```

Comment:
was there an intention here? The build failed to built this target with cordio sources. please review logs

Reply:
I see, this makes sense. So why it fails now then?